### PR TITLE
Add Scenarios 2-25, 27

### PR DESCRIPTION
**Pull request by jimjohnsonrollings**
JIRA: https://jira.dev.bbc.co.uk/browse/NEWSART-588

Adds more scenario data (note that there is no scenario 23, as this article has been withdrawn)

* [x] JIRA ticket added
* [x] Pull request URL added to JIRA ticket
* [x] Up-to-date with latest - `git pull --rebase origin latest`
* [ ] Runs locally - `npm run dev` & [http://localhost:7080/](http://localhost:7080/)
* [ ] Tests added for new features
* [ ] Tests pass - `npm test`
* [ ] E2E tests pass - `npm run test:e2e` (requires server running see [README](https://github.com/bbc/simorgh#tests) for details)

* [ ] Test engineer approval

### CODE REVIEW

File: data/scenario-03.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    "home": "http://www.bbc.co.uk/ontologies/passport/home/News",
+    "articleType": "news",
+    "genre": null
+  },
+  "publishedDateTime": "2018-06-04T00:00:00.000Z",
+  "lastUpdatedDateTime": "2018-06-05T12:00:00.000Z",
+  "seoHeadline": null,
+  "tags": {
+    "about": ["http://www.bbc.co.uk/things/3fbfd244-928b-498d-a585-9e39bd27fad3#id",
+
```
Comment (jtart) [inline data/scenario-03.json]: Remove this line

File: data/scenario-02.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "type": "image",
+    "aspectRatio": "1:1",
+    "relPath": "/cpsprodpb/439A/production/_100960371_syrians_and_asylum_v2-nc.png",
+    "copyrightHolder": "BBC",
+    "caption": "Praesent sagittis fringilla nibh ut vulputate. Ut _nec purus_ est. Vestibulum in eleifend nibh.",
+    "altText": "ICras dapibus ligula sit amet est aliquam, nec dignissim diam gravida."
+  }, {
+    "blockId": "da00008c-fc2b-44eb-8eae-30d65b26bd70",
+    "type": "paragraph",
+    "content": "Donec ut fermentum risus, eu sodales mauris. Quisque This is <sup>superscript</sup> vitae est sed nisl tincidunt facilisis. Nec vulputate metus sagittis./n/nQuisque quis libero bibendum, feugiat felis ut, dignissim tortor. Mauris metus lectus, lacinia sit amet massa ut, interdum convallis ipsum."
```
Comment (jtart) [inline data/scenario-02.json]: Is `/n` meant to be a line-break?

Also, are we handling new paragraphs with a line-break? @benjaminhobbs

**Review by jtart — CHANGES_REQUESTED** [review]

File: data/scenario-02.json (hunk 2 above)
Comment (jamesbhobbs) [reply data/scenario-02.json]: We will ignore /n so we can strip it out or leave it as a test of out md parsing.

**Comment by jamesbhobbs** [comment]
CI tests fixed

**Comment by jamesbhobbs** [comment]
I'm merging this to make the CR of the next tidy up PR easier (smaller diff). NB @jimjohnsonrollings I think for the withdrawn article we should add a file which contains null instead of having no file. Can you do this in the subsequent PR?

**Pull request merged**